extractor/main.go

```diff
@@ -32,9 +31,9 @@ var (
 
 // Define extractor config
 type extractorCfg struct {
-	fileType  string
-	sourceDir string
```

Comment:
why is this name change needed?

Reply:
Sorry, working on another PR that will introduce this change. Is it okay to keep it in order to not have to resolve conflicts later?

Reply:
Leftover from another PR. Removed: [391d2ec](https://github.com/gnolang/tx-exports/pull/10/commits/391d2ecf46f86a829d1ddc2ce6792f20c03bad32)